tests/integration/targets/podman_pod/tasks/main.yml

```diff
@@ -347,6 +347,47 @@
         that:
           - pod15_b_info is not changed
 
+    # Reset the namespace check.
```

Comment:
Reviewers: I am unsure of what to do here. This looks like another bug with regards to idempotency, it doesn't seem `podman inspect` has any info that would allow figuring out which is which.

Reply:
Probably we should use `CreateCommand` to understand if `userns` was passed to Podman.

```json
"CreateCommand": [
            "podman",
            "pod",
            "create",
            "--name",
            "pod2",
            "--userns",
            "auto"
        ],
```
Like we do here for example: https://github.com/containers/ansible-podman-collections/blob/0ae6baac3d673da1d1d545f2e971152fc2b6a8d3/plugins/module_utils/podman/podman_container_lib.py#L1068C1-L1071
Maybe worth to open issue to Podman itself, looks like a bug in inspection for me.
Anyway, this change is OK.

Reply:
Yeah, I think it's only an option in such cases.
Of course it doesn't prevent you to submit issues upstream anyway :)